_includes/head.html

```diff
@@ -4,11 +4,15 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
   <meta name="description" content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 | escape }}">
-  <link rel="stylesheet" href="{{ "/assets/css/normalize.css" | prepend: site.baseurl_root }}">
-  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: site.baseurl_root }}">
+  <style>
+    {% capture styles %}
+      {% include {{ "/assets/css/normalize.css" | prepend: site.baseurl_root }} %}
```

Comment:
In order to use `{% include file_name.ext %}` the `file_name.ext` must be inside the `_includes` directory. So four things must happen
- [ ] Update the inclusion template

```html
  <style>
    {% capture styles %}
      {% include main.scss %}
      {% include normalize.css %}
    {% endcapture %}
    {{ styles | scssify }}
  </style>
```
- [ ] Update `_config.yml`
```yaml
sass:
  style: compressed
```

- [ ] Move `normalze.css` and `main.scss` inside `_includes` folder

- [ ] Remove the _front matter_ from `main.scss` which is 

```
---
remove me, and the lines above/below me
---
```